stories/tooltip.stories.js

```diff
@@ -37,9 +37,9 @@ stories.add('default', () => (
     </Box>
 
     <Box>
-      <Tooltip text="Hello!">
-        <Link href="#tooltip" onClick={e => e.preventDefault()}>
-          Link with tooltip
+      <Tooltip text="Eight action item have been accepted.">
```

Comment:
Nitpick but `item` should probably be pluralized.

Reply:
😮good catch!